src/main/resources/application.yml

```diff
@@ -30,6 +30,8 @@ janitor:
     migrate-upgrade: true
     quiet-down-timeout: 20s
     terminate-timeout: 5s
+    retention-check-interval: 1d
+    completed-flight-retention: 7d
```

Comment:
can we make this little longer? It took 30 days for Google to remove delete a project. It might be helpful if we can keep the light little longer than that.